terra/reports.py

```diff
@@ -261,6 +268,103 @@ def unit_report(unit, start_date=None, end_date=None):
     return calculate_totals(data)
 
 
+def get_treq_list(fund, start_date=None, end_date=None):
+    start_date, end_date = check_dates(start_date, end_date)
+    rows = Funding.objects.filter(
+        fund=fund, treq__departure_date__gte=start_date, treq__return_date__lte=end_date
+    ).values(travel=F("treq"))
+
+    rows2 = ActualExpense.objects.filter(
```

Comment:
@aprigge Pet peeve :) - generic numbered variables.  `rows` is OK generally, but once we get to `rows2` we should reconsider naming in this method.  `funding_rows` and `actualexpense_rows` or similar would help the reader better understand what data is being used and why.